examples/demo/src/types.ts

```diff
@@ -73,6 +73,7 @@ export interface FieldProps<T extends Record = Record> {
     record?: T;
     source?: string;
     basePath?: string;
+    resource: string;
```

Comment:
should be optional as it's injected by react-admin

Reply:
fixed in #4847